test/format/failure_configs/CONFIG.WT-10384

```diff
@@ -0,0 +1,163 @@
+[2023/01/15 23:27:10.720]     ############################################
```

Comment:
Can you remove the timestamp at the start of each line?

Reply:
Yes I will do that. My eyes simply didn't see it anymore.